tests/test_discovery.py

```diff
@@ -170,4 +134,5 @@ def test_run(self):
                          if item.get("breadcrumb", []) != []
                          and item.get("breadcrumb", ["properties", None])[1]
                          not in actual_automatic_fields}),
-                    msg="Not all non key properties are set to available in metadata")
+                    msg="Not all non key properties are set to available in metadata",
```

Comment:
Noticed that many cases have removed the msg string while others have both msg and logging.  Some of the msg strings seem more detailed and informative than the logging line but this one seems more redundant.  Is there a convention or is it just a case by case judgement on whether to keep one or both?

Reply:
I don't think the msg strings are helpful in most cases. I left them as-is in a few assertions where it may not be obvious at first glance what the failing assertion is supposed to be testing.